common/src/cellular_at_core.c

```diff
@@ -111,21 +105,21 @@ CellularATError_t Cellular_ATIsPrefixPresent( const char * pString,
     char * ptrPrefixChar = NULL;
     char * ptrChar = NULL;
 
-    if( pResult == NULL )
+    if( pResult == NULL || pString == NULL )
```

Comment:
Similar question with this function. Are separate pResult and atStatus parameters / return values really necessary?

Reply:
Yes, this change makes the CELLULAR_AT_SUCCESS meaningless.
Let me fix this changes.